src/kv/RocksDBStore.cc

```diff
@@ -593,6 +593,7 @@ RocksDBStore::~RocksDBStore()
 {
   close();
   delete logger;
+  logger = nullptr;
```

Comment:
what's the point of setting a pointer variable after deleting it? we don't reference it anymore.